src/client/Microsoft.Identity.Client/Instance/AadAuthority.cs

```diff
@@ -32,6 +33,13 @@ internal AadAuthority(AuthorityInfo authorityInfo) : base(authorityInfo)
 
         internal override string TenantId { get; }
 
+        internal bool IsWorkAndSchoolOnly()
```

Comment:
>IsWorkAndSchoolOnly [](start = 22, length = 19)

is this used anywhere? do we have a test for it?

Reply:
Oops, bad cherry pick, will undo it!